### ISSUE TITLE
[DEPRECATION WARNING]: Skipping task due to undefined attribute

### ISSUE THREAD

**Issue by ssbarnea** [DataDog/ansible-datadog#14]
With ansible 2.0 we get this warning while using the role:

```
TASK [Datadog.datadog : service] ***********************************************
ok: [atlas2.do.citrite.net] => {"changed": false, "enabled": true, "name": "datadog-agent", "state": "started"}

TASK [Datadog.datadog : service] ***********************************************
skipping: [atlas2.do.citrite.net] => {"changed": false, "skip_reason": "Conditional check failed", "skipped": true}

TASK [Datadog.datadog : Create a configuration file for each Datadog check] ****
[DEPRECATION WARNING]: Skipping task due to undefined attribute, in the future this will be a fatal error.. This feature will be removed in a future release. Deprecation warnings
can be disabled by setting deprecation_warnings=False in ansible.cfg.
```

**Comment by olivielpeau** [comment]
Thanks @ssbarnea for the feedback!

We're in the process of checking that the role works with Ansible 2.0, I'm adding your feedback to the list of things that we have to check.

**Comment by mnewell** [comment]
When I use this role to configure agent integrations for Postgres, Mongo, and Docker it works just fine. With RabbitMQ the generated YAML file gets munged inexplicably.

I send code like this to the role to configure RabbitMQ:

```yaml
  roles:
    - { role: Datadog.datadog, sudo: true }
    - datadog
  vars:
    datadog_api_key: "0000000000000"
    datadog_checks:
      rabbitmq:
        init_config:
        instances:
        - rabbitmq_api_url: https://hostname1.com:15672/api/
          rabbitmq_user: ********
          rabbitmq_pass: ********
          tags:
            - env:prod
            - prod_rmq4
            - prod_rabbitmq
          queues_regexes:
            - .*WebReportJobQueue.*
            - .*tx_history_queue.*
          vhosts:
            - prod
            - prodjob
```

And when I run it in a playbook, it generates an /etc/dd-agent/conf.d/rabbitmq.yaml like this:

```yaml
init_config: null
instances:
-   queues_regexes:
    - .*WebReportJobQueue.*
    - .*tx_history_queue.*
    rabbitmq_api_url: https://hostname1.com:15672/api/
    rabbitmq_pass: ********
    rabbitmq_user: ********
    tags:
    - env:prod
    - prod_rmq4
    - prod_rabbitmq
    vhosts:
    - prod
    - prodjob
```

Is there a bug or am I actually screwing up the config YAML?

**Comment by olivielpeau** [comment]
This has been fixed through #21 and released with `1.0.0`. Closing.

Thanks @ssbarnea!
